File: pegasusapp/src/models/comunicadoAlumnos.ts
import { CuadernoComunicados } from './cuadernoComunicados';
import { Usuario } from './usuario';

export interface ComunicadoAlumnos {
  id: number;
  id_Comunicado: number;
  id_Alumno?: number | null;

  comunicado?: CuadernoComunicados | null;
  alumno?: Usuario | null;
}

File: PegasusV1/PegasusV1/Entities/ComunicadoAlumnos.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace PegasusV1.Entities
{
    [Table("COMUNICADO_ALUMNOS")]
    public class ComunicadoAlumnos
    {
        public int Id { get; set; }
        public int? Id_Comunicado { get; set; }
        [ForeignKey("Id_Comunicado")]
        public CuadernoComunicados? Comunicado { get; set; }
        public int? Id_Alumno { get; set; }
        [ForeignKey("Id_Alumno")]
        public Usuario? Alumno {  get; set; }
    }
}
